config/bootstrap.php

```diff
@@ -216,7 +216,8 @@
 /*
  * Only try to load DebugKit in development mode
  * Debug Kit should not be installed on a production system
+ * You may also set autoRun to false and append ?debug=true to your URLs to toggle DebugKit on.
  */
 if (Configure::read('debug')) {
-    Plugin::load('DebugKit', ['bootstrap' => true]);
+    Plugin::load('DebugKit', ['bootstrap' => true, 'autoRun' => true]);
```

Comment:
I don't think this actually does anything. Instead I think you want autoRun set in Configure.